src/main/java/net/minestom/server/event/player/PlayerPluginMessageEvent.java

```diff
@@ -46,6 +46,7 @@ public byte[] getMessage() {
      * @return the message
      */
     @NotNull
+    @Deprecated(forRemoval = true)
```

Comment:
Why is this deprecated?

Reply:
Ok, I will keep it and simply add a note in the javadoc